bindings/python/src/alert.cpp

```diff
@@ -441,7 +463,7 @@ void bind_alert()
 
     class_<listen_succeeded_alert, bases<alert>, noncopyable>(
         "listen_succeeded_alert", no_init)
-        .def_readonly("endpoint", &listen_succeeded_alert::endpoint)
```

Comment:
the reason I ask is because it's a lot more scalable to make sure the one converter for this type works, and then just bind the members, than to write a separate getter function for every type that uses it.